blocks/hero-banner/hero-banner.js

```diff
@@ -83,4 +83,7 @@ export default function decorate($block) {
   if ($contentRow) {
     decorateTextContent($contentRow, $block);
   }
+  if ($block.classList.contains('overlap')) {
```

Comment:
One concern here not related to your change though. Why we have started using variable starting with $ symbol. i think we should be consistent because everywhere we have used normal variables.

Reply:
Good catch. Maybe we can refactor the existing code.